src/pages/AccountDetails/AccountTokensTable/components/AccountTokensTableHeader.tsx

```diff
@@ -0,0 +1,95 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+
+import { Pager, TableSearch } from 'components';
+import { TokenTypeEnum } from 'types';
+
+export interface AccountTokensTableHeaderUIType {
+  tokenCount?: number;
+}
+
+export const AccountTokensTableHeader = ({
+  tokenCount = 0
+}: AccountTokensTableHeaderUIType) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const { type } = Object.fromEntries(searchParams);
+
+  const updateTokenType = (typeValue?: TokenTypeEnum) => {
```

Comment:
this could be transformed into 
```
const updateTokenType = (typeValue?: TokenTypeEnum) => () => {
...
}
```

and use it like this: `<Element onClick={updateTokenType(value)} />` or `<Element onClick={updateTokenType()} />`

Reply:
updated